backend/routes/api/user.js

```diff
@@ -0,0 +1,99 @@
+const express = require("express");
+const router = express.Router();
+const User = require("../../models/User");
+
+// get all users
+router.get("/", async (req, res) => {
```

Comment:
Awesome! Love how you're following RESTful guidelines here.

Reply:
😃